bcipy/simulator/helpers/artifact.py

```diff
@@ -0,0 +1,79 @@
+""" Handles artifacts related logic ie. logs, save dir creation, result.json, ..."""
+import datetime
+import logging
+import os
+import sys
+from typing import Optional
+
+from bcipy.config import ROOT
+
+# For a simulation, two loggers are configured: a top level logger summarizing
+# setup and progress, and a logger for each simulation run. The root logger is
+# re-configured for each simulation run.
+TOP_LEVEL_LOGGER_NAME = 'sim_logger'
+DEFAULT_LOGFILE_NAME = 'sim.log'
+DEFAULT_SAVE_LOCATION = f"{ROOT}/data/simulator"
+
+
+def configure_logger(log_path: str,
```

Comment:
I suggest we use the configure_logger already present in system_utils and modify it to meet this use case. Here, it seems that fmt and configuring two handlers are necessary?

Reply:
Added to https://www.pivotaltracker.com/story/show/188328498.

One of the handlers logs to the file and the other logs to stdout, so the user has feedback on the terminal regarding the progress.